libraries/botbuilder-runtime/test/settings/base.json

```diff
@@ -8,7 +8,11 @@
     "bad": false,
     "ok": "howdy"
   },
+  "numbers": {
+    "bad": "uh oh",
+    "ok": 10
+  },
```

Comment:
Is the delta in base.json still necessary? The PR's initial test additions were removed.

Reply:
Nope - good catch.